app/helpers/shared_helper/ab_list_helper.rb

```diff
@@ -1,135 +1,25 @@
 module SharedHelper::AbListHelper
   private
 
-  def shared_button_summary(record, sb_items, resolve, custom_button)
+  def shared_button_list_summary(record)
     summary = [
-      shared_button_basic_info(record, sb_items, custom_button),
-      object_details(record, resolve),
-      object_attribute(record, resolve),
-      visibility_expression(record),
-      attribute_value_pair(resolve),
-      visibility(custom_button, sb_items)
+      shared_button_list_basic_info(record)
     ]
     safe_join(summary)
```

Comment:
we can bring in the contents of `shared_button_list_basic_info` into `shared_button_list_summary` and then delete the method `shared_button_list_basic_info`